src/bentoml/_internal/service/loader.py

```diff
@@ -172,7 +169,7 @@ def recover_standalone_env_change():
         object.__setattr__(instance, "_import_str", f"{module_name}:{attrs_str}")
         return instance
     except ImportServiceError:
-        if sys_path_modified and working_dir:
+        if working_dir:
```

Comment:
we probably don't need this check neither now?

Reply:
I think this check is ensuring that if the assignment fails it doesn't try to remove `None`.

Reply:
Make sense, I'd suggest we move the working_dir assignment L75-81 out of the `try` block and remove this `if working_dir` check as well

Reply:
@sauyon made a quick patch, could you help take a look?